solution_template/createUiDefinition.json

```diff
@@ -183,6 +183,26 @@
               ]
             },
             "visible": true
+          },
+          {
+            "name": "jdkType",
+            "type": "Microsoft.Common.DropDown",
+            "label": "JDK Type",
+            "defaultValue": "Zulu",
+            "toolTip": "Azul Systems Zulu Enterprise builds of OpenJDK is recommended for Azure. If you change your mind, you can always SSH into your VM and install a different SDK.",
```

Comment:
JDK?

Reply:
typo

Reply:
I mean the last work: "you can always SSH into your VM and install a different ~~SDK~~JDK. "?